src/app/routes/enrolment/utils/format-validity-period-string.ts

```diff
@@ -1,5 +1,5 @@
 import { TimeDurationPipe } from 'src/app/shared/pipes/time-duration/time-duration.pipe';
 export const formatValidityPeriod = (num: number): string => {
```

Comment:
why you're transforming it here instead using pipe in html?

Reply:
How do I just call the transform method in the pipe in the html. I don't want a date, I want the hours, seconds, etc.

Reply:
pipe have a `transform` method which changes variable/property/object into something more readable. It is very easy to test and reuse. Here is link: https://angular.io/guide/pipes
To use this pipe in html all you need is:
`{{ defaultValidityPeriod | timeDuration }}`.

Reply:
Ah sorry, I was looking at time shift pipe! Thanks.